slapp/transforms/transform_pipeline.py

```diff
@@ -113,6 +107,28 @@ class TransformPipelineSchema(argschema.ArgSchema):
                      "A value of -1 results in "
                      "using multiprocessing.cpu_count()")
     )
+    scale_offset = argschema.fields.Int(
+        required=False,
+        default=3,
+        description=("number of pixels scale corner is offset from "
+                     "lower left"))
+    full_scale_offset = argschema.fields.Int(
+        required=False,
+        default=12,
+        description=("number of pixels scale corner is offset from "
+                     "lower left"))
```

Comment:
"...in the full-field-of-view ROI outline."

Reply:
implemented